runtimes/pythonrt/pythonrt_test.go

```diff
@@ -129,6 +140,26 @@ func newCallbacks(svc *pySvc) *sdkservices.RunCallbacks {
 	return &cbs
 }
 
+func setupServer(l *zap.Logger) *http.Server {
+	ConfigureWorkerGRPCHandler(l)
+	server := &http.Server{
+		Addr:    "localhost:9980",
+		Handler: h2c.NewHandler(Server, &http2.Server{}),
+	}
+
+	if err := ConfigureLocalRunnerManager(l, LocalRunnerConfig{WorkerAddress: "localhost:9980"}); err != nil {
+		panic(err)
```

Comment:
No panic, pass `t *testing.T` to this setup and call `t.Fatal`

Reply:
done